tests/functional/test_logger_utils.py

```diff
@@ -51,142 +50,134 @@ def service_name():
+    # WHEN configuration copied from powertools logger to ALL external loggers
+    # AND external loggers used
     utils.copy_config_to_registered_loggers(source_logger=powertools_logger)
+    msg = "test message1"
     logger_1.info(msg)
     logger_2.info(msg)
     logs = capture_multiple_logging_statements_output(stdout)
 
-    # THEN
+    # THEN all external loggers used Powertools handler, formatter and log level
     for index, logger in enumerate([logger_1, logger_2]):
         assert len(logger.handlers) == 1
-        assert type(logger.handlers[0]) is logging.StreamHandler
-        assert type(logger.handlers[0].formatter) is formatter.LambdaPowertoolsFormatter
+        assert isinstance(logger.handlers[0], logging.StreamHandler)
+        assert isinstance(logger.handlers[0].formatter, formatter.LambdaPowertoolsFormatter)
         assert logger.level == log_level.INFO.value
         assert logs[index]["message"] == msg
         assert logs[index]["level"] == log_level.INFO.name
 
 
 def test_copy_config_to_ext_loggers_include(stdout, logger, log_level):
-
-    msg = "test message"
-
-    # GIVEN a external logger and powertools logger initialized
+    # GIVEN an external logger and powertools logger initialized
     logger = logger()
     powertools_logger = Logger(service=service_name(), level=log_level.INFO.value, stream=stdout)
 
-    # WHEN configuration copied from powertools logger to ALL external loggers AND our external logger used
+    # WHEN configuration copied from powertools logger to INCLUDED external loggers
+    # AND our external logger used
     utils.copy_config_to_registered_loggers(source_logger=powertools_logger, include={logger.name})
+    msg = "test message2"
     logger.info(msg)
     log = capture_logging_output(stdout)
 
-    # THEN
+    # THEN included external loggers used Powertools handler, formatter and log level.
     assert len(logger.handlers) == 1
-    assert type(logger.handlers[0]) is logging.StreamHandler
-    assert type(logger.handlers[0].formatter) is formatter.LambdaPowertoolsFormatter
+    assert isinstance(logger.handlers[0], logging.StreamHandler)
+    assert isinstance(logger.handlers[0].formatter, formatter.LambdaPowertoolsFormatter)
     assert logger.level == log_level.INFO.value
     assert log["message"] == msg
     assert log["level"] == log_level.INFO.name
 
 
 def test_copy_config_to_ext_loggers_wrong_include(stdout, logger, log_level):
-
-    # GIVEN a external logger and powertools logger initialized
+    # GIVEN an external logger and powertools logger initialized
     logger = logger()
     powertools_logger = Logger(service=service_name(), level=log_level.INFO.value, stream=stdout)
 
-    # WHEN configuration copied from powertools logger to ALL external loggers AND our external logger used
+    # THEN included external loggers used Powertools handler, formatter and log level.
```

Comment:
Shouldn't it be something like `WHEN configuration copied from powertools logger to INCLUDED NON EXISTING external loggers`?